demos/connector/neo4j/undirected-graphsage-on-cora-neo4j-example.ipynb

```diff
@@ -0,0 +1,1117 @@
+{
```

Comment:
<p>I think we need more explicit descriptions/explanation before this cell to talk about how to set up a neo4j instance to run this notebook and how to adjust the parameters you're passing into the <code>get_node_data_from_neo4j</code> method depending on your setup. It's probably good to assume that the user has installed stellargraph and knows nothing else, but is interested to learn how to run this notebook. You could also include some helpful links to external documentation instead of having to explain everything since it might be a lot to explain</p><p> </p><p>Also minor thing, but the print happens <em>after</em> all the vertices have been loaded, so could be better phrased as "loaded" instead of "loading"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/799/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Some of this could be done in a README file instead too